src/v1_services/chat_service.py

```
@@ -0,0 +1,279 @@
+    → 4단계: 검색 결과를 f-string으로 프롬프트에 삽입
+    → 5단계: Gemini에 프롬프트 전달하여 답변 생성
+    → 6단계: API 응답 반환
+
+[v1의 한계점]
+- 매 요청마다 vertexai.init()과 모델 로드를 반복하여 비효율적
+- 한국어 질문을 번역 없이 그대로 임베딩하여 영어 법률과의 검색 정확도가 낮음
+- 임베딩/검색/프롬프트/LLM이 하나의 함수에 몰려있어 유지보수가 어려움
+"""
+
+import os
+from typing import List, Dict, Any
```

Ruff (0.15.2)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)